pkg/ipamd/ipamd.go

```diff
@@ -861,7 +903,7 @@ func (c *IPAMContext) tryAssignPrefixes() (increasedPool bool, err error) {
 			err = c.awsClient.AllocIPAddresses(eni.ID, 1)
 			if err != nil {
 				ipamdErrInc("increaseIPPoolAllocIPAddressesFailed")
-				return false, errors.Wrap(err, fmt.Sprintf("failed to allocate one IPv4 prefix on ENI %s, err: %v", eni.ID, err))
+				return false, errors.Wrap(err, fmt.Sprintf("failed to allocate one IP addresses on ENI %s, err: %v", eni.ID, err))
```

Comment:
why changed IPv5 prefix to IPAdderss?

Reply:
ohh..missed while copying. Let me fix this.